xtask/src/task_phd.rs

```diff
@@ -41,9 +41,25 @@ macro_rules! cargo_warn {
 }
 
 pub(crate) fn cmd_phd(phd_args: Vec<String>) -> anyhow::Result<()> {
+    let mut arg_iter = phd_args.iter().map(String::as_str).peekable();
+
+    let meta = cargo_metadata::MetadataCommand::new()
+        .no_deps()
+        .exec()
+        .context("Failed to run cargo metadata")?;
+    let phd_dir = relativize(&meta.target_directory).join("phd");
+
+    let mut tmp_dir = phd_dir.join("tmp");
+    let now = time::SystemTime::now();
+
+    if arg_iter.peek() == Some(&"tidy") {
```

Comment:
Should this command and a `--no-tidy` (rather than just the implied ENV option) option be moved into the arg parsing logic, especially so it can be visible via --help and such?

Reply:
Yeah, that's not a bad idea. We probably ought to do that for the `--release` flag that builds `propolis-server` in release mode, as well.

Reply:
Okay, 1a5e641fa4efdd5ab80a63319005cc06e40bdd53 changes the PHD subcommand to handle its own arguments using `clap`. This does have the minor downside that we no longer forward `cargo xtask phd help` directly to `phd-runner`, so I've added an additional `runner-help` subcommand for printing `phd-runner`'s help, which seems like an acceptable compromise...